exercises/resolving-merge-conflict/MaryHadALittleLamb.md

```diff
@@ -2,31 +2,34 @@
 
 Mary had a little lamb,
 Little lamb, little lamb.
-Mary had a little sheep,
-Its fleece was white as coal.
+Mary had a little lamb,
+Its fleece was white as snow.
 
 And everywhere that Mary went,
 Mary went, Mary went,
 Everywhere that Mary went,
-Like a diamond in the sky.
 
-It followed her to office one day,
-Office one day, office one day.
-It followed her to office one day,
+The lamb was sure to go.
+
+It followed her to school one day,
+School one day, school one day.
+It followed her to school one day,
 Which was against the rules.
 
-When the blazing sun is gone,
-When he nothing shines upon,
-Then you show your little light,
-Twinkle, twinkle, all the night.
+It made the children laugh and play,
+Laugh and play, laugh and play.
+It made the children laugh and play
+To see a lamb at school.
+
```

Comment:
Why extra line here?

Reply:
Got it. I didn't check the feedbacks yet actually. But I will from now

Reply:
Removed extra lines at end. The blank lines in middle are separation for paragraphs of poem.